ext/nnstreamer/tensor_converter/tensor_converter_flatbuf.cc

```diff
@@ -108,12 +110,14 @@ fbc_convert (GstBuffer *in_buf, GstTensorsConfig *config, void *priv_data)
     std::string _name = tensor->Get (i)->name ()->str ();
     const gchar *name = _name.c_str ();
 
-    config->info.info[i].name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
-    config->info.info[i].type = (tensor_type) tensor->Get (i)->type ();
+    _info = gst_tensors_info_get_nth_info (&config->info, i);
+
+    _info->name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
```

Comment:
What if _info->name is already allocated?

Reply:
Internally tensor-converter calls convert() callback after initializing the tensors-config struct.
To clearly release tensors-config, I will append init code in this callback.
```
  gst_tensors_config_free (config);
  gst_tensors_config_init (config);
```

Reply:
updated. plz review the changes again.

Reply:
Although gsttensor_converter.c::2429 calls ```gst_tensor_config_init``` before calling ```->convert```, in case someone call this function from another context (this is a callback available to other modules), let's simply add ```g_free(_info->name)``` before line 117, which will do nothing if _info->name is NULL as supposed.

Reply:
updated